### ISSUE TITLE
[Feature Request] Ability to drag and drop to other apps

### ISSUE THREAD

**Issue by RayBB** [picturama/picturama#30]
Thanks so much for making this app. It's a pretty cool project and seems to work quite well :)
Anyway, I've been trying to find a simple photo management tool for my parents on Linux and this seems like it's a good way for them to easily scroll and see all their photos on the computer.

I was wondering if it's possible for you to add the ability to drag and drop photos to other apps (specifically the webbrowser).
Basically, it's hard for them to click "browse" and then navigate to a photo. But if they could see the photo in your app and then just drag it to the browser window for auto upload that would work so great.
Not sure if this is an easy ask based on how you've developed the app but thought I'd ask.

Thanks!

**Comment by til-schneider** [comment]
Yes, dragging images out of Picturama would be possible (for the technically interested see [Electron articel about native file drag & drop](https://electronjs.org/docs/tutorial/native-file-drag-drop)).

But I don't really understand the original use case. Why would your parents want to drag images into a browser in the first place? Do they want to see them full screen? This would be easier using the detail view by double clicking an image in Picturama.

**Comment by RayBB** [comment]
Here's their use case.
Imagine they took a bunch of photos on a family vacation. They currently use Picasa to look at the photos and crop them, maybe do some light editing. Now they want to go and post a photo to Facebook. When they click "browse for files" they see a menu of a bunch of folders and then have to navigate to the correct folder (easier said than done). Once in the right folder they see tiny thumbnails of dozens of similar photos (imagine taking a bunch of photos of a group trying to make sure everyone's eyes are open) and have to struggle to see which one is the one they wanted when it was bigger. Sure they could write down the file name but that's not so easy when they're already not computer savvy. 

What would be great is if they could scroll through your program and once they see the photo they like they could simply drag and drop over to the photo upload page. 

I hope this is a bit easier of an explanation.

**Comment by til-schneider** [comment]
Thank you for the explanation. I think, this would be a nice feature.

The only concern I still have is whether users might get confused as they may think that images could be dropped somewhere inside Picturama - like reordering images within a section or moving images to other sections.

**Comment by JoseAlessioSilva92** [comment]
i would like this option, but in my case is when i edit pictures on photoshop and illustrator.

**Comment by til-schneider** [comment]
For editing photos, maybe the "version feature" may be the right thing. See #22.

**Comment by akjoshi** [comment]
This would be a nice feature to have - drag and drop the picture from picturama to external app and browser.

Also, provide the configuration options: a)  checkbox to remove all exif info by default if dropped to browser. b) default file size if dropped to browser  (ex: small/medium/large/original for sharing purpose).  

P.S. 'Export' feature with the option to reduce size and remove exif data is one of the most valuable features of this organizer.

**Comment by til-schneider** [comment]
I think, all these default options are a bit too indirect: When dragging fotos there would be no logical connection to the default settings, so the behaviour could be unexpected to the user if he doesn't remember the settings.

But your suggestion brings me to another idea: In the export dialog there could be a drag zone (e.g. a stack of photos) which could be dragged to other applications. So the user could: Select some photos, open export, change export settings, drag the stack of photos to another application (instead of pressing the export button).

**Comment by akjoshi** [comment]
@til-schneider I like your suggestion. Since 'export' feature already exists, the drag and drop could be built on the export dialog. Good thing about this option is that users can choose privacy setting and file size on every drag and drop operation.
